src/measure_theory/borel_space.lean

```diff
@@ -554,10 +552,10 @@ begin
   exact continuous_dist continuous_fst continuous_snd
 end
 
-lemma measurable_dist {α : Type*} [metric_space α] [second_countable_topology α]
+lemma measurable.dist {α : Type*} [metric_space α] [second_countable_topology α]
```

Comment:
When you rename a lemma like that (which is definitely a good idea), it means that `measurable_dist'` can also be renamed to `measurable_dist`. Same thing for many lemmas below.